COPY/usr/bin/manageiq-db-ready

```diff
@@ -6,41 +6,43 @@ require "manageiq-password"
 
 $stdout.sync = true
 
-def wait_for_service(host, port)
-  loop do
-    `ncat #{host} #{port} </dev/null 2>/dev/null`
-    break if $?.success?
-
-    puts "#{host} #{port} - not accepting connections"
-    sleep(10)
-  end
-
-  puts "#{host} #{port} - accepting connections"
+def service_ready?(host, port)
+  `ncat #{host} #{port} </dev/null 2>/dev/null`
+  $?.success?
```

Comment:
There's probably a cleaner way to do this since we don't care about the output, but this is fine for now since it was the original code.